resource/css/skosmos.css

```diff
@@ -5,6 +5,7 @@
   --medium-color: #dfe5ed;
   --secondary-medium-color: #b8c9db;
   --light-color: #f2f5f7;
+  --red-color: #a0270a;
```

Comment:
I think `alert-color` would be more "semantic" - it's possible someone wants to use completely different colors.